view_outputs.py

```diff
@@ -4,7 +4,7 @@
 from absl import app, flags, logging
 
 import windtunnel
-from windtunnel import postprocessing
+from windtunnel import WindTunnelOutputs
```

Comment:
Maybe follow the recommendation to not import classes directly:
https://google.github.io/styleguide/pyguide.html#22-imports